lib/coherence/controllers/unlock_controller.ex

```diff
@@ -49,25 +49,26 @@ defmodule Coherence.UnlockController do
       case LockableService.unlock_token(user) do
         {:ok, user} ->
           if user_schema.locked?(user) do
-            if Config.mailer?() do
-              send_user_email :unlock, user, router_helpers().unlock_url(conn, :edit, user.unlock_token)
-              put_flash(conn, :info, Messages.backend().unlock_instructions_sent())
-            else
-              put_flash(conn, :error, Messages.backend().mailer_required())
+            conn = case Config.mailer?() do
```

Comment:
I see three issues in this block. First, I would prefer that an if statement be used here since there only to execution blocks and the condition is a simple equality check which doesn't require a pattern match.

My second point is that I believe you can replace the assignment to `conn` with a pipeline. 

The third point (which becomes mute when point 2 is addressed) is that I prefer to have the block to the right of the assignment put on a separate with one indentation level as I have seen correctly done in previous code in this PR :)